custom/opm/reports.py

```diff
@@ -1057,7 +1057,8 @@ def awc_data(self):
         # populate awcs with cases from that awc
         for case_object in case_objects:
             awc = case_object.awc_name
-            awcs[awc] = awcs.get(awc, []) + [case_object]
+            awc_uid = case_object.owner_id
+            awcs[awc_uid] = awcs.get(awc_uid, []) + [case_object]
```

Comment:
I like this idea of using owner_id as a proxy for awc_name.  I think we might need to use that in the CMR and BPR too, otherwise I'd think the reports might not agree.  Then we could also use the `user_data_as_hierarchy` util to filter cases by owner id, similar to what I specified [here](https://github.com/dimagi/commcare-hq/commit/165fb987e588c1aea68327d8a5823214e4d9cf97), but without needing to do the full user traversal (this is still a full traversal, but at least it's not of entire user docs).